draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -122,13 +122,15 @@ For example, for Slow Start, as specified in {{!RFC5681}}, limit(maxFS)=2*maxFS,
 cwnd_new = cwnd + min (N, SMSS)
 cwnd = min(cwnd_new, 2*maxFS)
 ~~~
+where cwnd and SMSS follow their definitions in {{!RFC5681}} and N is the number of previously unacknowledged bytes acknowledged in the incoming ACK.
 
```

Comment:
"incoming" or "received"?

Reply:
See my suggestion (open in browser I think, since it seems you commented on the old text, not the new?!)